woocommerce/woocommerce
# Add override loop template code snippet

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Add the "Override loop template and show quantities next to add to cart buttons." code snippet.

Closes https://github.com/woocommerce/woodocs/issues/97.

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Check the added code snippet.
2. Verify that it respects the [coding standards](https://github.com/woocommerce/woodocs/blob/trunk/quality-and-best-practices/coding-standards.md).

### Changelog entry

n/a

<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
